styles.css

```diff
@@ -7,356 +7,355 @@
-    flex-wrap: wrap;
-    justify-content: flex-start;
-    list-style: none;
-}
-nav ul li{
-    padding:15px 15px ;
-}
-nav ul li:hover{
-    background-color: #83624a;
-    border-radius: 15px;
-}
-nav ul li.active{
-    /* background-color: #83624a; */
-    border-bottom: 4px solid bisque;
-}
-nav ul li a{
-    color: bisque;
-    text-decoration: none;
-    font-family: 'Patrick Hand';
-    font-size: 20px;
+:root {
+  min-height: 100vw;
+  max-width: 100vw;
+  margin: 0 auto;
+}
+nav {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  background-color: #553117;
+}
+.navBar img {
+  height: 50px;
+  width: 50px;
+  border-radius: 50%;
+  border: 2px solid orange;
+  padding: 8px;
+}
+nav .logo {
+  display: flex;
+  flex-direction: row;
+  padding: 10px;
+  padding-top: 12px;
+}
+nav .logo .title {
+  font-family: 'Oleo Script Swash Caps';
+  font-size: 24px;
+  padding: 8px;
+  padding-top: 18px;
+  padding-left: 20px;
+  color: bisque;
+}
+nav ul {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  list-style: none;
+}
+nav ul li {
+  padding: 15px 15px;
+}
+nav ul li:hover {
+  background-color: #83624a;
+  border-radius: 15px;
+}
+nav ul li.active {
+  /* background-color: #83624a; */
+  border-bottom: 4px solid bisque;
+}
+nav ul li a {
+  color: bisque;
+  text-decoration: none;
+  font-family: 'Patrick Hand';
+  font-size: 20px;
 }
 
 /* =============== MEDIA QUERY ================= */
 
-@media all and (max-width:800px) {
-    nav{
-        justify-content: flex-start;
-    }
-    nav ul{
-        justify-content: space-between;
-
-    }
-    /* .classicCoffee{
+@media all and (max-width: 800px) {
+  nav {
+    justify-content: flex-start;
+  }
+  nav ul {
+    justify-content: space-between;
+  }
+  /* .classicCoffee{
         padding: 20px;
         display: flex;
         flex-wrap: wrap;
         row-gap: 20px;
     } */
-    /* .buy{
+  /* .buy{
         text-align: right;
         justify-content: flex-end;
         display: flex;
         flex-direction: column;
     } */
 
-    /* .titleAbout{
+  /* .titleAbout{
         padding-left: 0px;
     } */
 }
-@media all and (max-width:600px) {
-    nav{
-        flex-direction: column;
-        align-items: center;
-    }
-    nav ul{
-        width: 100vw;
-        flex-direction: column;
-        align-items: stretch;
-        text-align: center;
-        overflow: hidden;
-        width: 100%;
-    }
-    .classicCoffee{
-        padding: 20px;
-        display: flex;
-        justify-content: center;
-        flex-basis: 400px;
-        flex-wrap: wrap;
-        row-gap: 20px;
-        column-gap: 20px;
-    }
+@media all and (max-width: 600px) {
+  nav {
+    flex-direction: column;
+    align-items: center;
+  }
+  nav ul {
+    width: 100vw;
+    flex-direction: column;
+    align-items: stretch;
+    text-align: center;
+    overflow: hidden;
+    width: 100%;
+    padding: 0px;
```

Comment:
The <ul> element has a native padding applied to the left of the element which you need to reset to 0px for the element to sit in the middle of the nav container. By adding `padding: 0px`, the default padding is reset and the element comes right in the middle.